src/app/core/services/auth.service.ts

```diff
@@ -64,7 +57,7 @@ export class AuthService {
     });
   }
 
-  getUser(){
+  getUser() {
     return this.user$;
   }
 
```

Comment:
let's try to define types for UserMetadata - the backend should have a good schema for it now